cves/kernel/CVE-2014-0101.yml

```diff
@@ -56,6 +51,7 @@ description_instructions: |
   Your target audience is people just like you before you took any course in
   security
 description:
+  CVE-2014-0101 exposes a critical vulnerability in the Linux kernel's SCTP protocol implementation, specifically within the 'sctp_sf_do_5_1D_ce' function responsible for authenticating incoming SCTP packets. Attackers can remotely target systems running susceptible versions of the Linux kernel by crafting malicious SCTP packets. When received by the vulnerable system, these packets exploit the flawed validation process in 'sctp_sf_do_5_1D_ce', causing memory corruption and, consequently, a system crash due to a NULL pointer dereference. This vulnerability affects systems using outdated or unpatched Linux kernel versions, posing a serious risk of remote denial of service.
```

Comment:
Wow, this seems very severe since all the attacker needs to do is sending malicious SCTP packets to induce a crash

Reply:
yep i the vulnerability rating was severe for this one